lineage/figures/figure11.py

```diff
@@ -0,0 +1,80 @@
+""" This file depicts the distribution of phase lengths versus the states. """
+import numpy as np
+import pandas as pd
+from matplotlib.ticker import MaxNLocator
+import seaborn as sns
+
+from ..Analyze import run_Analyze_over
+from ..data.Lineage_collections import Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM
+from .figureCommon import getSetup, subplotLabel
+
+data = [Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM, Lapatinib_Control, Lapt25uM, Lapt50uM, Lap250uM]
+
+lpt_g1 = []
+lpt_g2 = []
+lpt12_g1 = [] # for fig. 12
+lpt12_g2 = [] # for fig. 12
+
+gmc_g1 = []
+gmc_g2 = []
+gmc12_g1 = [] # for fig. 12
+gmc12_g2 = [] # for fig. 12
+
+# Run fitting
+output = run_Analyze_over(data, np.repeat([4, 3], 4))
+gemc_tHMMobj_list = [oo[0] for oo in output[0: 4]]
+gemc_states_list = [oo[1] for oo in output[0: 4]]
+lapt_tHMMobj_list = [oo[0] for oo in output[4: 8]]
+lapt_states_list = [oo[1] for oo in output[4: 8]]
+
+for idx, lapt_tHMMobj in enumerate(lapt_tHMMobj_list):
+    for lin_indx, lin in enumerate(lapt_tHMMobj.X):
+        for cell_indx, cell in enumerate(lin.output_lineage):
+            cell.state = lapt_states_list[idx][lin_indx][cell_indx]
```

Comment:
Why would the cell state be different than what's in the list?

Reply:
cell states were not applied yet, I mean after fitting when printed cell states, they were `None`, so I had to assign them, but I think I don't need to necessarily assign them to `cell.state`, I can just use them...

Reply:
Yep.